docs/content/guides/non-argument-deps.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: Assets without I/O | Dagster
+description: Often we don't want to build or load our assets as Python objects in memory
+---
+
+# Assets without I/O
+
+In the tutorial, all the assets we've looked at are straightforward to work with as Python objects in memory - e.g. `topstories` is created by constructing a DataFrame of the top stories from the Hacker News API.
+
+However, often we don't want to build or load our assets as Python objects in memory. For example, we might compute an asset by copying a file, by issuing a `create table` statement to a database, or by executing a command-line utility.
+
+In those situations, we can define assets whose functions don't have return values or arguments. And we can mix those assets with assets that do have return values or arguments.
+
+---
+
+## Bringing in more stopwords
+
+In the [tutorial](/tutorial), we defined an asset that contains a list of stopwords to filter out common words or prepositions from our word cloud. The English language has a lot of stopwords, so rather than hardcoding them into your asset, you'll bring them in from a separate data source.
+
+You'll augment your asset graph with a new asset that contains the stopwords from a zip file on the internet and create an asset from those stopwords. These stopwords will replace the hardcoded stopwords in your `topstories_word_cloud` asset. By the end of this guide, your asset graph will have two new assets:
```

Comment:
🔧 It's looking like the user needs to complete the tutorial before they start this guide. If that's the case, then there should be a **Prerequisites** section that says so.